library/src/skins/default/src/sass/modules/tool/rubrics/_rubrics.scss

```diff
@@ -789,14 +787,32 @@ div.collapse-toggle-buttons > button {
     -webkit-filter: drop-shadow(2px 0px 0px #AAA);
 }
 
+.arrow-0 {
+    bottom: 42px;
+    right: -18px;
+}
+
+.arrow-comment {
+    bottom: 50px;
+    right: -18px;
+}
+
 .label-rubrics {
     font-weight: bold;
 }
 
-.rubrics-popover {
-    top: 389px; 
-    left: -10px; 
+.rubrics-popover, .rubrics-comment-popover {
     display: none; 
     position: absolute; 
     padding: 10px;
 }
+
+.rubrics-popover {
+    top: 389px; 
+    left: -10px; 
+}
+
+.rubrics-comment-popover {
+    top: -50px !important; 
```

Comment:
Do you need this because of toggleEditor in sakai-rubric-grading-comment.js? top is being set in that file.

Reply:
I'll take a look.